README.md

```diff
@@ -27,6 +27,14 @@ being sent, or a patch is unlandable using `lando`, please
 
 > **NOTE** This tool doesn't currently support mercurial
 
+## Quick tutorial
+To set up phlay, enter the following in the terminal:
+```
+echo 'export PATH="$HOME/githubcode/phlay:$PATH"' >> ~/.bash_profile
```

Comment:
This is a bit overly specific to your personal setup, where `githubcode` is where phlay has been cloned. I think I'd prefer to not have a specific tutorial.